src/styl/now-value.styl

```diff
@@ -0,0 +1,22 @@
+@import '_variables'
+
+.dollert-now-value-container
+  padding-bottom: 15px
+  text-align: center
+  line-height: 20px
+  background $color-white
+  border-bottom 1px solid $color-grey
+  .dollar-now-value-label
```

Comment:
These classes don't need to be 2 levels specific. So, you don't need keep them nested to `dollert-now-value-container`